packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -138,6 +141,12 @@ export const RemoteServerFactory = {
         // This is the ORIGIN under which we are serving the page.
         // because it's an origin, it does not end with a slash.
         options.appOrigin = process.env.APP_ORIGIN = `${options.protocol}://${options.appHostname}`
+
+        if ('MRT_ALLOW_COOKIES' in process.env) {
+            // Toggle cookies being passed and set
+            options.ssrAllowCookies = process.env.MRT_ALLOW_COOKIES?.toLowerCase() === 'true'
```

Comment:
@bredmond-sf actually I think there's a miss here. Should this be...?

```suggestion
            options.ssrAllowCookies = process.env?.MRT_ALLOW_COOKIES?.toLowerCase() === 'true'
```

Reply:
Disregard... you've got the check on `145` at the open of the `if` block